s3prl/downstream/a2a-vc-vctk/README.md

```diff
@@ -0,0 +1,92 @@
+# S3PRL-VC: open-source voice conversion framework with self-supervised speech representations
+
+Development: [Wen-Chin Huang](https://github.com/unilight) @ Nagoya University (2021).  
+If you have any questions, please open an issue, or contact through email: wen.chinhuang@g.sp.m.is.nagoya-u.ac.jp
+
+**Note**: This is the **any-to-any** recipe. For the **any-to-one** recipe, please go to the [a2o-vc-vcc2020](../a2o-vc-vcc2020/) recipe.
+
+## Task
+
+In this downstream, we focus on training any-to-any (A2A) voice conversion (VC) models.
+We perform model training on the **VCTK** corpus, and evaluate on the _intra-lingual VC_ task in **voice conversion challenge 2020 (VCC2020)**.
+For more details about the intra-lingual VC task and the VCC2020 dataset, please refer to the original paper:
+
+- Yi, Z., Huang, W., Tian, X., Yamagishi, J., Das, R.K., Kinnunen, T., Ling, Z., Toda, T. (2020) Voice Conversion Challenge 2020 –- Intra-lingual semi-parallel and cross-lingual voice conversion –-. Proc. Joint Workshop for the Blizzard Challenge and Voice Conversion Challenge 2020, 80-98, DOI: 10.21437/VCC_BC.2020-14. [[paper](https://www.isca-speech.org/archive_v0/VCC_BC_2020/pdfs/VCC2020_paper_13.pdf)] [[database](https://github.com/nii-yamagishilab/VCC2020-database)]
+
+
+## Implementation
+
+We only provide the config for the **Taco2-AR** model. It is essentialy a modified, attention-free Tacotron2 model. For the speaker embedding, we used a [d-vector](https://static.googleusercontent.com/media/research.google.com/zh-TW//pubs/archive/41939.pdf) implementation by the [Resemblyzer](https://github.com/resemble-ai/Resemblyzer). For the vocoder, we only offer the [Hifi-GAN](https://arxiv.org/abs/2010.05646), implemented in the [open source project](https://github.com/kan-bayashi/ParallelWaveGAN) developed by [kan-bayashi](https://github.com/kan-bayashi).
+
+## Dependencies:
+
+- `parallel-wavegan`
+- `fastdtw`
+- `pyworld`
+- `pysptk`
+- `jiwer`
+- `resemblyzer`
+
+You can install them via the `requirements.txt` file.
+
+## Usage
+
+### Preparation
+```
+# Download the VCTK and the VCC2020 datasets.
+cd <root-to-s3prl>/s3prl/downstream/a2a-vc-vctk
+cd data
+./vcc2020_download.sh vcc2020/
+./vctk_download.sh ./
+cd ../
+
+# Download the pretrained vocoders.
+./vocoder_download.sh ./
+```
+
+### Training 
+The following command starts a training run given any `<upstream>`.
+```
+cd <root-to-s3prl>/s3prl/downstream/a2a-vc-vctk
+./downstream/a2a-vc-vctk/vc_train.sh <upstream> downstream/a2a-vc-vctk/config_ar_taco2.yaml <tag>
+```
+Along the training process, you may find converted speech samples generated using the Griffin-Lim algorithm automatically saved in `<root-to-s3prl>/s3prl/result/downstream/a2a_vc_vctk_taco2_ar_<upstream>/<step>/test/wav/`.
```

Comment:
This path can be misleading:
<root-to-s3prl>/s3prl/result/downstream/a2a_vc_vctk_taco2_ar_<upstream>/<step>/test/wav/

Please change to:
<root-to-s3prl>/s3prl/result/downstream/a2a_vc_vctk_<tag>_<upstream>/<step>/test/wav/